### DISCUSSION TITLE
point_0735

### DISCUSSION THREAD

**Discussion by andrea29-star** [cloudsen12/dataset#347] [Roy]
"20200501T015701_20200501T020437_T52SFB"
![image](https://user-images.githubusercontent.com/54947195/147801702-a22ec53f-7ad2-476c-9e87-5da0484bb601.png)

"20200416T015649_20200416T015648_T52SFB"
![image](https://user-images.githubusercontent.com/54947195/147801758-729f74a4-a6a2-43c7-9304-f9a15d22b409.png)

@ryali93

**Comment by AntoniCesar** [comment]
20200521T015701_20200521T015658_T52SFB
![Captura](https://user-images.githubusercontent.com/86502862/153803457-62a670cb-6c19-42b7-a879-32bf767bbd4e.JPG)
